app/controllers/admins/company_values_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Admins
+  class CompanyValuesController < AdminController
+    def index
+      @company_values = CompanyValue.all
```

Comment:
It's good idea to avoid passing instance variables to views, one of the biggest issues is global access. [Visuality has good article about it](https://www.visuality.pl/posts/should-i-use-instance-variables-in-rails-views)

To pass it to view in safer way you can do it like this:
```suggestion
      render :index, locals: { company_values: CompanyValue.all}
```
then in view you can just use `company_values` instead of `@company_values`